AmazonChimeSDK/AmazonChimeSDK/internal/audio/DefaultAudioClientObserver.swift

```diff
@@ -197,6 +198,60 @@ class DefaultAudioClientObserver: NSObject, AudioClientDelegate {
         }
     }
 
+    public func transcriptEventsReceived(_ events: [Any]?) {
+        if events == nil {
+            return
+        }
+
+        guard let internalEvents = events as? [TranscriptEventInternal] else {
+            return
+        }
+
+        internalEvents.forEach { rawEvent in
+            ObserverUtils.forEach(observers: transcriptEventObservers) { (observer: TranscriptEventObserver) in
+                if let rawStatus = rawEvent as? TranscriptionStatusInternal {
+                    let status = TranscriptionStatus(type: Converters.Transcript.toTranscriptionStatusType(type: rawStatus.type),
+                                                     eventTimeMs: rawStatus.eventTimeMs,
+                                                     transcriptionRegion: rawStatus.transcriptionRegion,
+                                                     transcriptionConfiguration: rawStatus.transcriptionConfiguration,
+                                                     message: rawStatus.message)
+                    observer.transcriptEventDidReceive(transcriptEvent: status)
```

Comment:
do we need to give it single everytime?

Reply:
Yes, the wire protocol was designed to be able to send a list of events to support historical transcripts, which was removed at a later stage. We kept the wire protocol plural and updated client callback to only provide a single event at a time, since backend is sending single event.